app/presenters/creator_home_presenter.rb

```diff
@@ -66,11 +66,19 @@ def creator_home_props
       end
     end
 
-    tax_forms = (Time.current.year.downto(seller.created_at.year)).each_with_object({}) do |year, hash|
-      url = seller.eligible_for_1099?(year) ? seller.tax_form_1099_download_url(year: year) : nil
-      hash[year] = url if url.present?
+    tax_center_enabled = Feature.active?(:tax_center, seller)
+    if tax_center_enabled
```

Comment:
Conditional logic under the new feature flag to properly show the "Your 1099 tax form for is ready!" notice on dashboard.